### PR TITLE
Docs: Update connector's options

### LINKED ISSUE #2252: Add `connector-chromium` documentation

### LINKED ISSUE #2471: [docs] Issue with 'Chrome'
https://webhint.io/docs/user-guide/connectors/connector-chrome/index.html

The example at the bottom of the above document appears to be incorrect:

```
{
    "connector": {
        "name": "chrome",
        "options": {
            "defaultProfile": true,
            "useTabUrl": false,
            "tabUrl": "https://empty.webhint.io/",
            "flags": ["--headless", "--disable-gpu"],
            "waitForContentLoaded": 10000
        }
    },
    ...
}
```

Correct me if I'm wrong but doing that was giving me invalid configuration errors. Should it be something like this? Flatten `options`

```
{
    "connector": {
        "name": "chrome",
        "defaultProfile": true,
        "useTabUrl": false,
        "tabUrl": "https://empty.webhint.io/",
        "flags": ["--headless", "--disable-gpu"],
        "waitForContentLoaded": 10000
    },
    ...
}
```

I noticed this trying to set `flags`.

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- ~~Added/Updated related tests.~~

## Short description of the change(s)

Fix #2471
Fix #2252

### CODE REVIEW

**Comment by molant** [comment]
macOS is failing because chrome released a new version and the cask hasn't been updated yet. I've [submitted a PR for that](https://github.com/Homebrew/homebrew-cask/pull/63630)

File: packages/hint/docs/user-guide/concepts/connectors.md [hunk 1]
```diff
@@ -53,104 +41,12 @@ with the values you want to modify:
...
-document details the known differences or issues among the official
-connectors.
-
-### jsdom
-
-* It will not send the events for:
-
-  * `element::#document`
-  * `element::#comment`
+Please check check the [dedicated page][connectors] for each one to know
```
Comment (sarvaje) [inline packages/hint/docs/user-guide/concepts/connectors.md]: 'check' is duplicated

Comment (molant) [reply packages/hint/docs/user-guide/concepts/connectors.md]:
```suggestion
Please check the [dedicated page][connectors] for each one to know
```

Comment (molant) [reply packages/hint/docs/user-guide/concepts/connectors.md]: Fixed

File: packages/connector-jsdom/README.md [hunk 2]
```diff
@@ -37,28 +37,38 @@ configuration file:
...
 
-### jsdom configuration
+## Options
 
-`jsdom` allows you to configure the following:
+The set of settings supported by jsdom connector are:
 
-* `headers`: the headers used to fetch the resources. By default they are:
+* `ignoreHTTPSError (boolean)`: Indicates if errors with certificates
+  should be ignored. Use this when checking self-signed certificated.
```
Comment (antross) [inline packages/connector-jsdom/README.md]:
```suggestion
  should be ignored. Use this when checking self-signed certificates.
```

File: packages/connector-chrome/README.md [hunk 3]
```diff
@@ -35,37 +35,31 @@ configuration file:
...
-* `flags? (string[])`: Allows you to pass in additional Chrome
-  command line API flags. Useful if you would like to start your
-  session in headless mode or with GPU disabled. Here's the full list
-  of [available command line flags][cli flags].
-  `['--no-default-browser-check']` is the default value.
-* `waitForContentLoaded (number)`: Time in milliseconds to wait for the
-  `loadingFinished` event from the `debugging protocol` before requesting
-  the body of a response. The default value is `10000` (10 seconds).
+* `ignoreHTTPSError (boolean)`: Indicates if errors with certificates
+  should be ignored. Use this when checking self-signed certificated.
```
Comment (antross) [inline packages/connector-chrome/README.md]:
```suggestion
  should be ignored. Use this when checking self-signed certificates.
```

File: packages/connector-local/README.md [hunk 4]
```diff
@@ -35,9 +31,42 @@ configuration file:
 }
 ```
 
+## Options
+
+`local` allows you to configure the following:
+
+* `pattern`: Add or ignore files defined in the pattern. By default the
+  `local` connector will use the following patter `['**', '!.git/**']`. This
```
Comment (antross) [inline packages/connector-local/README.md]:
```suggestion
  `local` connector will use the following pattern `['**', '!.git/**']`. This
```

File: packages/hint/docs/user-guide/concepts/connectors.md [hunk 5]
```diff
@@ -17,27 +17,15 @@ make it look like the following:
...
 Where `connectorName` is the name of the connector.
 
-## Built-in connectors and platform support
+## Official connectors and platform support
 
 All the built-in `connector`s run in any of the supported platforms:
 Linux, macOS, and Windows. The only caveat is that, for the `connector`
 that you specify in the`.hintrc` file, you will need to have the
 browser the `connector` is for installed as `webhint` will not
 install it for you.
```
Comment (antross) [inline packages/hint/docs/user-guide/concepts/connectors.md]: Seems this won't apply to any connectors other than `connector-puppeteer` in the near future. Think we should remove or reword this section?

File: packages/connector-puppeteer/README.md [hunk 6]
```diff
@@ -61,6 +62,22 @@ The set of settings supported by the Puppeteer connector are:
...
+
+* `auth`: The credentials and elements to authenticate on a website.
+  See next section for further details.
+* `browser (chrome|chromium|edge)`: Tells the preferred browser to
+  use. Webhint will search the executable for the given one and fail
+  if it does not find one. Keep in mind that not all browsers are
+  available in all platforms and that you need to **manually install
+  the browser**.
+* `ignoreHTTPSError (boolean)`: Indicates if errors with certificates
+  should be ignored. Use this when checking self-signed certificated.
```
Comment (antross) [inline packages/connector-puppeteer/README.md]:
```suggestion
  should be ignored. Use this when checking self-signed certificates.
```

File: packages/hint/docs/user-guide/concepts/connectors.md (hunk 5 above)
Comment (molant) [reply packages/hint/docs/user-guide/concepts/connectors.md]: Maybe when we deprecate `connector-chrome`?

Comment (antross) [reply packages/hint/docs/user-guide/concepts/connectors.md]: Seems reasonable.

**Comment by molant** [comment]
@antross @sarvaje changes are up

**Pull request merged**